### ISSUE TITLE
Unable to access jekyll website hosted at 127.0.0.1:4000 served through Bash [Windows 10 Insider build #14951]

### ISSUE THREAD

**Issue by keeratsingh** [Microsoft/BashOnWindows#1285]
- A brief description
  I am unable to access a jekyll website hosted at 127.0.0.1:4000 served through Bash [Windows 10 Insider build #14951]
  It says the host refused the connection.
  This works on a previous build #14393 as tested by rolling back to a previous build.
- Expected results
  The website is hosted and accessible at 127.0.0.1:4000
- Actual results (with terminal output if applicable)
  It says the host refused the connection when trying to access the website through a web browser
  Configuration file: /mnt/d/OneDrive/Projects/keeratsingh.github.io/_config.yml
          Source: source
     Destination: _site
  Incremental build: disabled. Enable with --incremental
      Generating...
                    done in 5.621 seconds.
  Auto-regeneration: enabled for 'source'
  Configuration file: /mnt/d/OneDrive/Projects/keeratsingh.github.io/_config.yml
  Server address: http://127.0.0.1:4000/
  Server running... press ctrl-c to stop.
- Your Windows build number
  14951
- Steps / All commands required to reproduce the error from a brand new installation 
  Build a jekyll website by:
  jekyll serve --force_polling

**Comment by benhillis** [comment]
@keeratsingh Could you check your firewall to see if port 4000 allows incoming connections?

**Comment by keeratsingh** [comment]
@benhillis 
Here is an output from the cmd prompt
C:\WINDOWS\system32>telnet 127.0.0.1 4000
Connecting To 127.0.0.1...Could not open connection to the host, on port 4000: Connect failed

- I have even tried setting an inbound rule from the firewall, it still doesn't open the port
- The same problem exists with build 14959 as well. Given below is the screenshot as seen the jekyll site is build and hosted at 127.0.0.1:4000
- netstat -an | find "4000" shows no 4000 port open

![jekyll](https://cloud.githubusercontent.com/assets/2684055/20028131/64d9997a-a2e6-11e6-9ca9-0e866295f05c.jpg)

**Comment by keeratsingh** [comment]
- Jekyll site build and hosted at 127.0.0.1:4000 works if I use the Jekyll on windows package through command prompt and not bash as seen in the screenshot below.
- netstat -an | find "4000" shows ports open and listening, which is not seen in case of Bash.

![jekyll2](https://cloud.githubusercontent.com/assets/2684055/20042163/00dbec4e-a42a-11e6-975d-2cefd9137741.jpg)

**Comment by sunilmut** [comment]
@keeratsingh - Thanks for trying out WSL.
It's working fine for me on build 14955 (I haven't tried on previous build and I don't see any reasons why it shouldn't be working on previous builds as well). Anyways, I used the sample Jekyll site from [here](https://github.com/maciakl/Sample-Jekyll-Site) for testing (Note that I have no affiliation to it). 

Steps I followed:
1. git clone the above repo
2. Jekyll serve

![jekyll](https://cloud.githubusercontent.com/assets/16765049/20160812/a0372c7e-a69e-11e6-886a-8accda1ec969.png)
